dedshit/blutter-termux
# code cleanup, revert comment and add new flag

- by @truefae #104
- revert base & heap_base comment
- needs latest r2 from git
- add flag (`--nu`) to disable update check
- fix creating visual studio solution
- add flag (`--ida-fcn`) to only generate IDA function names
- improve logic for checking `marking_stack_block_offset`, fixing issue with some dev versions
- Optimize snapshot analysis by reusing pre-built binaries
